CMakeLists.txt

```diff
@@ -132,6 +132,7 @@ option(USE_SYSTEM_EIGEN_INSTALL
 option(USE_TENSORRT "Using Nvidia TensorRT library" OFF)
 option(USE_ZMQ "Use ZMQ" OFF)
 option(USE_ZSTD "Use ZSTD" OFF)
+option(USE_CBLAS_IN_MKLDNN "Use CBLAS in MKLDNN" OFF)
```

Comment:
- Sorry about the naming -- Could you change it again to `USE_MKLDNN_CBLAS`? I checked the other options, and most of their names follow this pattern.
- Could you make this an dependent option?

```cmake
cmake_dependent_option(
  USE_MKLDNN_CBLAS "USE CBLAS in MKLDNN" ON
  "USE_MKLDNN" OFF)
```

Reply:
Thanks for your suggestion and I have revised it. Please help review. Thanks.